djstripe/models.py

```diff
@@ -459,7 +461,7 @@ def sync_current_subscription(self, cu=None):
         if sub:
             try:
                 sub_obj = self.current_subscription
-                sub_obj.plan = djstripe_settings.plan_from_stripe_id(sub.plan.id)
+                sub_obj.plan = sub.plan.id
```

Comment:
Since we're using models now, CurrentSubscription should leverage a ForeignKey. In that case, sub.plan.id should be converted into a Plan using some kind of sync utility (in case a plan exists in stripe that doesn't exist locally, kind of like a get_or_create)